.github/workflows/linux_meson.yml

```diff
@@ -102,3 +102,42 @@ jobs:
       run: |
         export OMP_NUM_THREADS=2
         python dev.py -n -j 2
+
+  test_venv_install:
+    name: Pip install into venv
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Setup Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+
+    - name: Install Ubuntu dependencies
+      run: |
+        # We're not running the full test suite here, only testing the install
+        # into a venv is working, so leave out optional dependencies. That's
+        # also why we can get away with an old version of OpenBLAS from Ubuntu
+        sudo apt-get update
+        sudo apt-get install -y libopenblas-dev pkg-config gfortran
+
+    - name: Create venv, install SciPy
+      run: |
+        python -m venv ../venvs/scipy-venv
```

Comment:
The third-party `virtualenv` tool I use does have subtle differences vs. the standard library `venv`--that may be worth checking as far as gh-16300 goes. Obviously, we can't test all these things in CI. For my purposes, if `venv` works and `virtualenv` doesn't I can probably just switch for now, though an eventual resolution is probably desirable if my problems aren't specific to my machine.

Reply:
Ugh yes, that's such a mess ... `venv`, `virtualenv`, `virtualenv-wrapper`, and who knows what other flavor. IIUC `venv` (stdlib) is now recommended. I'm not sure that's the issue though, I'll try to check.

Reply:
Virtualenvwrapper is simply a convenience wrapper around vanilla virtualenv, no behavior changes between these two at least.